modules/openapi-generator/src/main/resources/Java/libraries/rest-assured/api.mustache

```diff
@@ -85,7 +85,9 @@ public class {{classname}} {
     {{/isDeprecated}}
     public class {{operationIdCamelCase}}Oper {
 
+        public static final String REQ_METHOD = "{{httpMethod}}";
         public static final String REQ_URI = "{{path}}";
+        public static final String SUMMARY = "{{{summary}}}";
```

Comment:
@viclovsky what's the use case for this constant storing the summary (which is included in the javadoc string above)?

Reply:
Constants are necessary to fill report. 
Rest-assured is mostly client for testing. Tests often has integration with test report (such as [Allure](https://github.com/allure-framework/allure2)). Test report has detailed information about tests run results, curl arguments, command line etc.

Reply:
@viclovsky thanks for the explanation 👍

Reply:
It also intended to do the same with tags, but I haven't found easy way to do it.